legal-api/src/legal_api/core/filing.py

```diff
@@ -520,6 +520,12 @@ def get_document_list(business,  # pylint: disable=too-many-locals disable=too-m
                 adds = [FilingMeta.get_all_outputs(business.legal_type, doc) for doc in legal_filings]
                 additional = set([item for sublist in adds for item in sublist])
 
+                # Supress Certificate of Dissolution for Admin Dissolution
+                if filing.filing_type == Filing.FilingTypes.DISSOLUTION.value and \
+                        filing.storage.filing_sub_type == 'administrative' and \
+                        filing.storage.submitter_roles == UserRoles.staff.value:
+                    additional.remove('certificateOfDissolution')
```

Comment:
can you move this logic to `alter_outputs` function (line no 529)

Reply:
I did